ntbs-service/Pages/Notifications/Edit/_SpecimenDetails.cshtml

```diff
@@ -2,18 +2,14 @@
 
 @if (Model.Count() > 0)
 {
-    <h2>Summary by specimen</h2>
+    <h4>Summary by specimen</h4>
     <div id="specimens-details">
         @foreach (var specimen in Model)
         {
             <div class="notification-specimen-summary">
-                <h3>Specimen #@specimen.ReferenceLaboratoryNumber</h3>
+                <h4 class="section-header-with-background">Specimen #@specimen.ReferenceLaboratoryNumber</h4>

```

Comment:
It doesn't seem right that it's the same heading level as the "Summary by specimen" heading. We should be able to just make this a `h5`, no?